packages/gatsby-plugin-typescript/src/gatsby-node.js

```diff
@@ -19,7 +19,10 @@ module.exports.modifyWebpackConfig = ({ config }, { compilerOptions }) => {
   const opts = { compilerOptions: copts, transpileOnly: true }
   config.loader(`typescript`, {
     test,
-    loader: `ts-loader?` + JSON.stringify(opts),
+    loaders: [
+      `babel?${JSON.stringify(config._loaders.js.config.query)}`,
```

Comment:
I know it's a bit overkill, I just want to avoid any futur missed babel transformer (we never think to typescript module ^^).

I'm not a big fan with another package for babel, we have already manyyyyy packages (and it introduce some tricky part for anyone want use gatsby without starter)